plugin/core/transports.py

```diff
@@ -7,7 +7,6 @@
 from queue import Queue
 from typing import Any, Callable, Dict, Generic, IO, Protocol, TypeVar
 import http
-import json
```

Comment:
Should be fine to always import it and remove the import from the `except` block.